packages/zowe-explorer/package.json

```
@@ -192,6 +192,11 @@
         "title": "%listDataSets%",
         "category": "Zowe Explorer"
       },
+      {
+        "command": "zowe.ds.setDatasetFilter",
```

(nitpick) The above command is `listDataSets`, establishing the same capitalization here would make it easier for developers to pick up on the same naming scheme for related commands.


```suggestion
        "command": "zowe.ds.setDataSetFilter",
```

Find/replace should be easiest to cover all command references in one fell swoop.